CONTRIBUTING.md

```diff
@@ -1,23 +1,30 @@
-## Questions about Kodi?
+### Introduction
 
-To get your questions answered, please ask in the [Kodi community forums] (http://forum.kodi.tv/) or on IRC: #kodi on freenode.net
+**Kodi** uses github for development only, i.e. for pull requests and the review of such code.  
+**Do not open** an issue on github for your questions or bug reports.
+**Do not comment** on a Pull request at github unless you are involved in the testing of such or have something meaningful to contribute.
 
-Do not open an issue.
+#### Questions about Kodi?
 
-## Issue Reports
+To get your questions answered, please ask in the [Kodi community forums] (http://forum.kodi.tv/) or on **IRC:** [#kodi](http://webchat.freenode.net?nick=kodi-user&channels=%23kodi&prompt=1&uio=OT10cnVlde) on freenode.net
 
-Kodi uses github for development only, i.e. for pull requests and the discussion of code.
+#### Issue/Bug reports and discussions
 
-So we use a hook script to automatically close new issue created by you.
+Issue/Bug reports are created and reviewed at http://trac.kodi.tv using the kodi.tv forums username and password.
 
-If you can, we encourage you to investigate the issue yourself and create a Pull Request for us to review.
+We use a hook script to automatically close a related trac issue created by you if a specific Pull request addresses proposes a fix by a developer.
```

Comment:
It's not proper English ("addresses proposes") and it's not 100% accurate because the functionality does not apply to pull requests but to commits. If a commit contains either `fixes #12345` or `closes #12345` the hook script will automatically close the trac ticket #12345. The commit doesn't have to be part of a PR or anything.